boards/xtensa/esp32/esp32-wrover-kit/src/esp32_bringup.c

```diff
@@ -259,6 +263,20 @@ int esp32_bringup(void)
     }
 #endif
 
+#ifdef CONFIG_SENSORS_BMP180
+  /* Try to register BMP180 device in I2C0 */
+
+  ret = board_bmp180_initialize(0, 0);
+
```

Comment:
Same here!

Reply:
BTW, there is not a fixed rule about this spacing, so I will merge your patch anyway. Normally I try to duplicate Mr. Greg style, for example: mm/shm/shmctl.c and drivers/loop/losetup.c